### ISSUE TITLE
CSS not working

### ISSUE THREAD

**Issue by Blueforcer** [awtrix/awtrix-hd#9]
CSS via tailwind and Stylesheet doesnt work in awtrix dev package 0.1.15

**Comment by padarom** [comment]
Possibly related to https://github.com/vitejs/vite/issues/1579.

The compiled UMD module is
```js
(function(global, factory) {
  typeof exports === "object" && typeof module !== "undefined" ? module.exports = factory(require("@awtrix/common"), require("vue")) : typeof define === "function" && define.amd ? define(["@awtrix/common", "vue"], factory) : (global = typeof globalThis !== "undefined" ? globalThis : global || self, global.AwtrixComponent = global.AwtrixComponent || {}, global.AwtrixComponent["example@0-1-0"] = factory(global.AwtrixCommon, global.Vue));
})(this, function(common, vue) {
  "use strict";
  var _sfc_main = common.defineComponent({
    mounted() {
      this.lock();
    }
  });
  var frontend_vue_vue_type_style_index_0_scoped_true_lang = "\n#app[data-v-32dd3081] {\n  background: red;\n}\n";
  const _withId = /* @__PURE__ */ vue.withScopeId("data-v-32dd3081");
  vue.pushScopeId("data-v-32dd3081");
  const _hoisted_1 = {id: "app"};
  vue.popScopeId();
  const _sfc_render = /* @__PURE__ */ _withId((_ctx, _cache, $props, $setup, $data, $options) => {
    return vue.openBlock(), vue.createBlock("div", _hoisted_1, vue.toDisplayString(_ctx.app.name) + "@" + vue.toDisplayString(_ctx.app.version), 1);
  });
  _sfc_main.render = _sfc_render;
  _sfc_main.__scopeId = "data-v-32dd3081";
  return _sfc_main;
});
```
The variable `frontend_vue_vue_type_style_index_0_scoped_true_lang ` which contains the style isn't used anywhere. Might need to write a custom Vite plugin to make use of this.

**Comment by padarom** [comment]
After tons of trying to figure out how Vite/rollup compiles the UMD module and not getting a result I figured out that rollup indeed just compiles a separate `style.css` file. Therefore I just included this style in a link tag when rendering the component, which now works.

Tested with multiple `<style>` tags within the `frontend.vue`, with scoped styles and with `lang="stylus"`.

See https://github.com/awtrix/awtrix-hd/pull/8/commits/1cf81bf1128045e48483cde46e103e85ae03fbfb
